prisma/schema.prisma

```diff
@@ -33,6 +33,16 @@ model Recipe {
   @@map("recipe")
 }
 
+model Image {
+  id Int @default(autoincrement()) @id
+  name String
+  key String
+  recipe Recipe @relation(fields: [recipeId], references: [id])
```

Comment:
Altho this would work, it's not the best approach to place relation to the recipe directly on the image entity while this can be reusable / attached to different resources in the future.
Please refer to the relations article that I've sent you.
Consider moving this relation on the Recipe side or add some kind of junction table.